### PR TITLE
WT-8370 Add non timestamped configurations to data validation stress tests

### PR DESCRIPTION
**Pull request by korteland**
Unfortunately the last pull request got merged after some other, incompatible changes, so the testing I did on Friday was no longer valid. See the ticket for an updated set of passing test runs.

Leaving logging enabled here causes the test to fail, so I'm going to raise a separate issue for that.

### CODE REVIEW

File: test/evergreen.yml [hunk 1]
```diff
@@ -3797,7 +3825,9 @@ buildvariants:
     - name: format-smoke-test
     - name: format-failure-configs-test
     - name: data-validation-stress-test-checkpoint
+    - name: data-validation-stress-test-checkpoint-no-timestamp
```
Comment (quchenhao) [inline test/evergreen.yml:3828]: alphabetize

Comment (korteland) [reply test/evergreen.yml]: Thanks, fixed in 23d2dcc

**Review by quchenhao — APPROVED** [review]
lgtm

File: test/checkpoint/workers.c [hunk 2]
```diff
@@ -59,8 +59,8 @@ create_table(WT_SESSION *session, COOKIE *cookie)
           "memory_page_max=64KB,log=(enabled=false),%s",
           kf, vf, lsm));
     else
-        testutil_check(
```
Comment (kommiharibabu) [inline test/checkpoint/workers.c:62]: Can you fix the comment above as we are turning off the logging for both timestamped and non-timestamped scenarios?

File: test/evergreen.yml [hunk 3]
```diff
@@ -2589,6 +2603,20 @@ tasks:
             ${test_env_vars|} ../../../tools/run_parallel.sh 'nice ../../../test/checkpoint/recovery-test.sh "${data_validation_stress_test_args}" WT_TEST.$t' 120
 
   - name: data-validation-stress-test-checkpoint-fp-hs-insert-s1
+    depends_on:
```
Comment (kommiharibabu) [inline test/evergreen.yml:2606]: The stress test options 3 and 5 are also applicable to non-timestamped tables. Can we include them also as part of this?

File: test/checkpoint/workers.c (hunk 2 above)
Comment (korteland) [reply test/checkpoint/workers.c:62]: thanks, fixed in a715739

File: test/evergreen.yml (hunk 3 above)
Comment (korteland) [reply test/evergreen.yml:2606]: thanks - i've included them in a715739. there's a successful patch build for these new commits in the ticket comments.

**Review by kommiharibabu — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**